### PR TITLE
Add dask

### PR DESCRIPTION
**Pull request by koverholt**

### CODE REVIEW

File: recipes/dask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - dask.bytes
+    - dask.dataframe
+    - dask.dataframe.tseries
+    - dask.delayed
+    - dask.diagnostics
+    - dask.store
+
+about:
+  home: http://github.com/dask/dask/
+  license: BSD
```
Comment (jakirkham) [inline recipes/dask/meta.yaml]: Please specify the license version.

File: recipes/dask/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  version: "{{ version }}"
+
+source:
+  fn: dask-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
+  md5: f111b837ae969a690cf23cd75add92e2
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/dask/meta.yaml]: Please change to `python setup.py install --single-version-externally-managed --record=record.txt`.

**Comment by jakirkham** [comment]
Couple comments. Once they are resolved this should be good to go.

**Comment by koverholt** [comment]
I made the two changes you suggested. Thanks!

**Comment by 183amir** [comment]
LGTM, thanks @koverholt

File: recipes/dask/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+about:
+  home: http://github.com/dask/dask/
+  license: BSD 3-Clause
+  summary: 'Minimal task scheduling abstraction'
+
+extra:
+  recipe-maintainers:
+    - mrocklin
+    - jcrist
```
Comment (jakirkham) [inline recipes/dask/meta.yaml]: Is it alright for both of you to be added as maintainers here, @mrocklin and @jcrist?

Comment (mrocklin) [inline recipes/dask/meta.yaml]: +1 from me

Comment (jcrist) [inline recipes/dask/meta.yaml]: +1 from me as well.

**Comment by jakirkham** [comment]
Thanks everyone.

**Comment by jakirkham** [comment]
@koverholt @jcrist @mrocklin, this is now in `conda-forge`. The feedstock/repo can be found [here]( https://github.com/conda-forge/dask-feedstock ). Packages will be available from the `conda-forge` channel [here]( https://anaconda.org/conda-forge/dask ).

Also, you should receive an email inviting all of you to join conda-forge. Once you accept, you will be added to the `dask` team, which will give you commit privileges on the feedstock. To sign up for other packages, simply add your GitHub handle and the bots will take care of the rest. 

Please remember that you need to fork the feedstock and PR changes from your feedstock back to the main one. This is required because all branches on the main feedstock will automatically deploy builds.

Thanks for working on `dask` and welcome to conda-forge. 😄

**Comment by koverholt** [comment]
@jakirkham, thanks for the information and a big thanks to the conda-forge team for the great setup.

> To sign up for other packages, simply add your GitHub handle and the bots will take care of the rest.

I'm not sure where to do this. I noticed that @jcrist was not automatically added to the `dask` team even though he is listed in the `recipe-maintainers`, how can we get that setup?

**Comment by mrocklin** [comment]
He probably needs to agree to join the organization

**Comment by koverholt** [comment]
Ah right, thanks!

**Comment by jakirkham** [comment]
Yeah, that's why I try to explain these on merging. 😉 It gets easier after doing a couple.

**Comment by pelson** [comment]
Welcome aboard guys. Glad to have dask in conda-forge! 👍

**Pull request merged**